.github/workflows/event.yml

```diff
@@ -0,0 +1,16 @@
+name: GitHub Event Handler
```

Comment:
If I didn't know the implementation details I would never realize this has anything to do with check-enforcer. We should consider adding some comment in these files that might indicate what events are being handled.

Reply:
That's because the original intention was for this to be extended for any github actions that need to handle github events, so that we can avoid multiple runners getting spun up for each event.

Reply:
I'd rather the file be event type agnostic, with `azure-sdk-actions` being the place to look to know what's handled. Otherwise if we add new event handlers we'd have to update all these files in the repos.